app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::V1::ReviewsController < Api::V1::BaseController
+
+  def index
+    @booking = booking.find(params[:id])
```

Comment:
you're on ReviewsController, if you pass params[:id] nyou're passing id of review

Reply:
it has to be booking_id

Reply:
@booking = Booking.find(params[:booking_id]) <<< Booking with capital B